src/lib/utils.js

```diff
@@ -181,6 +181,7 @@ function getRefinements(results, state) {
   forEach(state.numericRefinements, (operators, attributeName) => {
     forEach(operators, (values, operator) => {
       forEach(values, (name) => {
+        name = name + '';
```

Comment:
cc @Jerskouille Tests were failing somewhere because we were comparing name: 1 to name: '1'. Then I thought: name should be a string, no matter what since it's called "name". If at some point we want the value we can still add it.

What do you think?